src/pages/previous.js

```diff
@@ -0,0 +1,152 @@
+import NavBar from "../components/navBar";
+import Chevron from "../assets/chevron_down.svg";
+import { GET } from "../utils/fetch";
+import { useEffect, useState } from "react";
+
+async function getQuestions() {
+  const weekly_questions = await GET("weekly_questions");
+  const questions = [];
+  for (const value in weekly_questions) {
```

Comment:
What are you trying to do here? Does `.map` not work?

Reply:
Wasn't working but now I did a bit differently and worked